README.md

```diff
@@ -38,7 +38,7 @@ These scripts have been tested in a Docker image of the following distributions,
   pacman -S base-devel ccache clang cmake git lld ninja python3
   ```
 
-If you intend to compile with PGO, please ensure that you also have `bc`, `bison`,`flex`, and `libssl-dev` (or equivalent) installed as you will be building some Linux kernels.
+If you intend to compile with PGO, please ensure that you also have `bc`, `bison`,`flex`, and `libssl-dev` (or equivalent) installed as you will be building some Linux kernels. If you are building for PowerPC (which the script does by default), make sure `mkimage` is available as well; the package is `u-boot-tools` on Debian/Ubuntu.
 
```

Comment:
This should only be required if you're building a uImage, which is certainly something you _can_ do, but it might be simpler to build a zImage instead?

Reply:
Hmm, you are building a zImage. Not sure why that calls mkimage but perhaps I'm just Wrong About Things.

Reply:
When building `ppc44x_defconfig`'s `zImage` target (same as [our continuous integration](https://github.com/ClangBuiltLinux/continuous-integration/blob/master/driver.sh#L108)):

```
  WRAP    arch/powerpc/boot/cuImage.bamboo
  WRAP    arch/powerpc/boot/cuImage.sam440ep
  WRAP    arch/powerpc/boot/cuImage.sequoia
  WRAP    arch/powerpc/boot/cuImage.rainier
  WRAP    arch/powerpc/boot/cuImage.taishan
  WRAP    arch/powerpc/boot/cuImage.katmai
  WRAP    arch/powerpc/boot/cuImage.warp
  WRAP    arch/powerpc/boot/cuImage.yosemite
INFO: Uncompressed kernel (size 0x5fba00) overlaps the address of the wrapper(0x400000)
INFO: Fixing the link_address of wrapper to (0x600000)
INFO: Uncompressed kernel (size 0x5fba00) overlaps the address of the wrapper(0x400000)
INFO: Fixing the link_address of wrapper to (0x600000)
INFO: Uncompressed kernel (size 0x5fba00) overlaps the address of the wrapper(0x400000)
INFO: Fixing the link_address of wrapper to (0x600000)
INFO: Uncompressed kernel (size 0x5fba00) overlaps the address of the wrapper(0x400000)
INFO: Uncompressed kernel (size 0x5fba00) overlaps the address of the wrapper(0x400000)
INFO: Fixing the link_address of wrapper to (0x600000)
INFO: Uncompressed kernel (size 0x5fba00) overlaps the address of the wrapper(0x400000)
INFO: Fixing the link_address of wrapper to (0x600000)
INFO: Fixing the link_address of wrapper to (0x600000)
INFO: Uncompressed kernel (size 0x5fba00) overlaps the address of the wrapper(0x400000)
INFO: Uncompressed kernel (size 0x5fba00) overlaps the address of the wrapper(0x400000)
INFO: Fixing the link_address of wrapper to (0x600000)
INFO: Fixing the link_address of wrapper to (0x600000)
INFO: Uncompressed kernel (size 0x5fba00) overlaps the address of the wrapper(0x400000)
INFO: Fixing the link_address of wrapper to (0x600000)
INFO: Uncompressed kernel (size 0x5fba00) overlaps the address of the wrapper(0x400000)
INFO: Fixing the link_address of wrapper to (0x600000)
INFO: Uncompressed kernel (size 0x5eb29c) overlaps the address of the wrapper(0x400000)
INFO: Fixing the link_address of wrapper to (0x600000)
INFO: Uncompressed kernel (size 0x5fba00) overlaps the address of the wrapper(0x400000)
INFO: Fixing the link_address of wrapper to (0x600000)
"mkimage" command not found - U-Boot images will not be built
../arch/powerpc/boot/Makefile:406: recipe for target 'arch/powerpc/boot/uImage' failed
make[3]: *** [arch/powerpc/boot/uImage] Error 1
make[3]: *** Waiting for unfinished jobs....
"mkimage" command not found - U-Boot images will not be built
../arch/powerpc/boot/Makefile:418: recipe for target 'arch/powerpc/boot/cuImage.sam440ep' failed
make[3]: *** [arch/powerpc/boot/cuImage.sam440ep] Error 1
"mkimage" command not found - U-Boot images will not be built
../arch/powerpc/boot/Makefile:418: recipe for target 'arch/powerpc/boot/cuImage.ebony' failed
make[3]: *** [arch/powerpc/boot/cuImage.ebony] Error 1
"mkimage" command not found - U-Boot images will not be built
../arch/powerpc/boot/Makefile:418: recipe for target 'arch/powerpc/boot/cuImage.warp' failed
make[3]: *** [arch/powerpc/boot/cuImage.warp] Error 1
"mkimage" command not found - U-Boot images will not be built
../arch/powerpc/boot/Makefile:418: recipe for target 'arch/powerpc/boot/cuImage.bamboo' failed
make[3]: *** [arch/powerpc/boot/cuImage.bamboo] Error 1
"mkimage" command not found - U-Boot images will not be built
../arch/powerpc/boot/Makefile:418: recipe for target 'arch/powerpc/boot/cuImage.katmai' failed
make[3]: *** [arch/powerpc/boot/cuImage.katmai] Error 1
"mkimage" command not found - U-Boot images will not be built
../arch/powerpc/boot/Makefile:418: recipe for target 'arch/powerpc/boot/cuImage.rainier' failed
make[3]: *** [arch/powerpc/boot/cuImage.rainier] Error 1
"mkimage" command not found - U-Boot images will not be built
../arch/powerpc/boot/Makefile:418: recipe for target 'arch/powerpc/boot/cuImage.yosemite' failed
make[3]: *** [arch/powerpc/boot/cuImage.yosemite] Error 1
"mkimage" command not found - U-Boot images will not be built
../arch/powerpc/boot/Makefile:418: recipe for target 'arch/powerpc/boot/cuImage.taishan' failed
make[3]: *** [arch/powerpc/boot/cuImage.taishan] Error 1
"mkimage" command not found - U-Boot images will not be built
../arch/powerpc/boot/Makefile:418: recipe for target 'arch/powerpc/boot/cuImage.sequoia' failed
make[3]: *** [arch/powerpc/boot/cuImage.sequoia] Error 1
arch/powerpc/Makefile:296: recipe for target 'zImage' failed
make[2]: *** [zImage] Error 2
/root/tc-build/build/llvm/linux-5.1/Makefile:312: recipe for target '__build_one_by_one' failed
make[1]: *** [__build_one_by_one] Error 2
make[1]: Leaving directory '/root/tc-build/build/llvm/linux-5.1/out'
Makefile:169: recipe for target 'sub-make' failed
make: *** [sub-make] Error 2
```

Reply:
Oh ok, it must be a feature of the 32bit target/defconfig. I wonder if pmac32 would be a better one.

Reply:
Is this comment thread resolved or what? I'll resign from review (LGTM) but @daxtens should probably approve this PR.

Reply:
I personally think changing the defconfig that we build here should be done in our continuous integration repo first (the stuff we build here should mirror what we build there).

Reply:
I've looked at the bugs you mentioned elsewhere. I think the change is reasonable in light of them.

Acked-by: Daniel Axtens <dja@axtens.net>

Reply:
(I don't know how to ack/approve stuff in GitHub, lmk if I need to do something.)

Reply:
There should be an add review button down where it says "Review required" or you can click on the "Files changed" tab.

Reply:
Done!